### PR TITLE
Include compiling multi-jvm tests in PR validation

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

**Review by chbatey — APPROVED** [review]
Doh this is indeed a good idea :)

LGTM

**Review by patriknw — COMMENTED** [review]
I’m pretty sure that we have had multi-jvm:jvm compile done as part of test:compile, which is better because the contributors (and we) detect when compiling locally

**Comment by johanandren** [comment]
Ok, I'll look into that.

**Comment by patriknw** [comment]
I think the problem is this line: https://github.com/akka/akka/blob/master/project/MultiNode.scala#L65
It was changed because sbt 1.0 warns about deprecation, but the new syntax doesn't do the same: https://github.com/akka/akka/commit/fca07bb9768c94ff1f9dcd13d518e9e1a389d01e#diff-af635decd1119b811e9f09470553f1b1R63

**Comment by johanandren** [comment]
Hmmm, I can't see how the new syntax doesn't do the same, `<<=` was redefine by using other keys, which is now replaced by `:= ... key.value ...`. 

I did however find this: https://stackoverflow.com/questions/23445644/triggered-execution-in-sbt-0-13-x#26476160 which indicates it may be a sbt bug.

**Comment by johanandren** [comment]
Our friends on the sbt team says it should have been fixed: https://github.com/sbt/sbt/pull/2908

**Comment by johanandren** [comment]
Argh.

```Scala
val hello = taskKey[Unit]("hello")
// snip
lazy val multiJvmSettings = 
  // this works
  hello in MultiJvm := println("hello"),
  hello in MultiJvm := ((hello in MultiJvm).triggeredBy(compile in Test)).value,
  // but this doesn't
  compile in MultiJvm := ((compile in MultiJvm).triggeredBy(compile in Test)).value,
```

**Comment by johanandren** [comment]
Putting it last in `multiJvmSettings` works, woho!

File: project/MultiNode.scala [hunk 1]
```diff
@@ -100,7 +99,10 @@ object MultiNode extends AutoPlugin {
      Def.settings((compile in MultiJvm) := {
       (headerCreate in MultiJvm).value
       (compile in MultiJvm).value
-    }) ++ headerSettings(MultiJvm)
+    }) ++ headerSettings(MultiJvm) ++ Seq(
+      // only works if I put it here ¯\_(ツ)_/¯
+      compile in MultiJvm := ((compile in MultiJvm).triggeredBy(compile in Test)).value
```
Comment (chbatey) [inline project/MultiNode.scala]: I believe you!

**Review by chbatey — APPROVED** [review]
LGTM for the better fix

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**